kymatio/scattering2d/tests/test_tensorflow_scattering2d.py

```diff
@@ -0,0 +1,314 @@
+import os
+import io
+import tensorflow as tf
+from kymatio.scattering2d import Scattering2D
+import torch
```

Comment:
i'm pretty sure that `import torch` is unnecessary here, amirite? :D

Reply:
it is, because of torch.load() - cf #419

Reply:
oh, ok, nvm; thanks for clarifying!